spyder/plugins/tests/test_editor_introspection.py

```diff
@@ -58,8 +60,9 @@ def get_spyder_pythonpath(*args):
     editor.introspector.plugin_manager.close()
 
 
-@pytest.mark.skipif(os.environ.get('CI', None) is not None,
-                    reason="This test fails too much in the CI :(")
+@pytest.mark.slow
+@pytest.mark.skipif(PY2 or PYQT4,
```

Comment:
Last time: please remove `PY2` from here.

Reply:
I think the failures only happen in PyQt4.

Reply:
But it segfaulted on Py2 + PyQt5 too... ?

Reply:
I can't see it now because I rebased it, but I'm 99% sure both Python tests failed, the QtPy4 one with segfault and the QtPy5 just hung and stopped, and then I restarted the QtPy5 one and it segfaulted too. But if you're sure, I can resend it again...

Reply:
Yup, [here](https://travis-ci.org/spyder-ide/spyder/jobs/325618685) you go. I cancelled midway but it did indeed segfault.

Reply:
Ok, not prob. Thanks for trying :+1: